content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Getting started with Integrations
+kind: documentation
+aliases:
+  - /getting-started-integrations/
+  - /guides/getting-started-integrations/
+  - /guides/integrations/
+further_reading:
+- link: "https://learn.datadoghq.com/course/view.php?id=2"
+  tag: "Learning Center"
+  text: "Introduction to Datadog"
+- link: "/integrations/"
+  tag: "Integrations"
+  text: "Our full list of integrations"
+- link: "/graphing/"
+  tag: "Graphing"
+  text: "Now that you have your integrations set up, build graphs to visulize your data"
+- link: "/monitors/"
+  tag: "Alerts"
+  text: "Set up alerts on your data"
+---
+
+An integration at the highest level is when you assemble a unified system from units that are usually considered separately. At Datadog, you can use integrations to bring together all of the units of your infrastructure and gain insight into the unified system as a whole - you can see pieces individually and also how individual pieces are impacting the whole. Datadogs integrations use a Python class called a check to define the typical metrics most people would want to monitor for that system.
+
+* Over 250 integrations are [officially listed][1], and supported in our [integrations core][2] (and we're always adding more).
```

Comment:
```suggestion
* Over 250 integrations are [officially listed][1], and supported in the [integrations core][2] (and more are always being added).
```
I think the guides can take on a more relaxed tone than the rest of the docs, but we still shouldn't use we/our